pkg/run/run.go

```diff
@@ -44,14 +44,15 @@ func (b *DockerRunner) Run(config *api.Config) error {
 
 	opts := docker.RunContainerOptions{
 		Image:        config.Tag,
+		Entrypoint:   []string{"/bin/env"},
```

Comment:
make it a default where?

but now i'm wondering if doing this is also going to break images that don't have /bin/env for one reason or another.  (this originally came from @mfojtik, i'm just re-implementing his change because the original one had to be reverted)